components/wso2is.key.manager.core/src/main/java/org/wso2/is/key/manager/core/internal/KeyManagerCoreServiceComponent.java

```diff
@@ -55,6 +58,10 @@ protected void activate(ComponentContext cxt) {
         try {
             cxt.getBundleContext().registerService(AuthenticationHandler.class, new ExtendedISAuthHandler(), null);
             cxt.getBundleContext().registerService(ScopeValidator.class, new RoleBasedScopesIssuer(), null);
+            cxt.getBundleContext().registerService(Axis2ConfigurationContextObserver.class.getName(),
+                    new ReservedUserCreationObserver(), null);
```

Comment:
Create single new ReservedUserCreationObserver() and pass to both registerService method

Reply:
Fixed with d06bf14